app/main.py

```diff
@@ -1,13 +1,40 @@
+
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
+
+
+# format_linter_error(error=error)
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+    "errors":
+        [
+            format_linter_error(error=errors[0]),
+            format_linter_error(error=errors[1])
```

Comment:
The current implementation assumes that there are always at least two errors in the `errors` list. This can lead to an `IndexError` if the list has fewer than two elements. Consider iterating over the `errors` list to handle any number of errors dynamically.